packages/mongo/oplog_tests.js

```diff
@@ -180,6 +182,118 @@ process.env.MONGO_OPLOG_URL && Tinytest.addAsync(
   },
 );
 
+const defaultOplogHandle = MongoInternals.defaultRemoteCollectionDriver().mongo._oplogHandle;
+let previousMongoPackageSettings = {};
+
+async function oplogOptionsTest({
+  test,
+  includeCollectionName,
+  excludeCollectionName,
+  mongoPackageSettings = {}
+}) {
+  try {
+    previousMongoPackageSettings = { ...(Meteor.settings?.packages?.mongo || {}) };
+    if (!Meteor.settings.packages) Meteor.settings.packages = {};
+    Meteor.settings.packages.mongo = mongoPackageSettings;
+
+    const myOplogHandle = new MongoInternals.OplogHandle(process.env.MONGO_OPLOG_URL, 'meteor');
+    MongoInternals.defaultRemoteCollectionDriver().mongo._setOplogHandle(myOplogHandle);
+
+    const IncludeCollection = new Mongo.Collection(includeCollectionName);
+    const ExcludeCollection = new Mongo.Collection(excludeCollectionName);
+
+    const shouldBeTracked = new Promise((resolve) => {
+      IncludeCollection.find({ include: 'yes' }).observeChanges({
+        added(id, fields) { resolve(true) }
+      });
+    });
+    const shouldBeIgnored = new Promise((resolve, reject) => {
+      ExcludeCollection.find({ include: 'no' }).observeChanges({
+        added(id, fields) { 
+          // should NOT fire, because this is an excluded collection:
+          reject(false);
+        }
+      });
+      // we give it just 2 seconds until we resolve this promise:
+      setTimeout(() => {
+        resolve(true);
+      }, 2000);
+    });
+
+    // do the inserts:
+    await IncludeCollection.rawCollection().insertOne({ include: 'yes', foo: 'bar' });
+    await ExcludeCollection.rawCollection().insertOne({ include: 'no', foo: 'bar' });
+
+    test.equal(await shouldBeTracked, true);
+    test.equal(await shouldBeIgnored, true);
+  } catch(err) {
+    // we just have this catch to be able to use the finally! ;)
+    throw err;
```

Comment:
Nope, we don't 😛 
```suggestion
```

Reply:
Then you please have to help me! Otherwise I can't make the last test work, where I want to test that it fails if both settings are used at the same time. If I catch the error here, then I can't "listen" for an error in the actual test!

Reply:
I meant you can just `try { ... } finally { ... }`. Is there a problem with that?

Reply:
Allllright, my bad 😁 you are right, `try { ... } finally { ... }` works just fine! ✌️